Set up Resources section

Dindihub opened the pull request:
Set up and population of the TUF.io Resources section.

Please review @chalin

Dindihub committed:
homepage edits

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
homepage edits

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
edit homepage CNCF logo

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Update README.md

Dindihub committed:
correct CNCF logo size

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
resolve merge issues

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
can't remember what this commit does

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Update content/en/_index.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update content/en/_index.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update content/en/_index.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update hugo.yaml

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update hugo.yaml

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update hugo.yaml

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update hugo.yaml

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Merge branch 'main' into homepage

Dindihub committed:
rebase conflicts

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
changes in README

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Update README.md

Dindihub committed:
Style CNCF logo

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Update README.md

Dindihub committed:
Update README.md

Dindihub committed:
Fix format issues

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
resolve merge conflicts

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
fix merge conflicts

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
delete TUF png logo

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
edit code not resolved by rebase

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Working on site colors

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Update content/en/_index.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update content/en/_index.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update content/en/_index.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
edit pull from remote

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
styling edits on homepage

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
styling edits on homepage

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Set up resources page

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Structure resources section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Merge remote-tracking branch 'origin/main' into resources

Dindihub committed:
Add content to resources section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Update README.md

chalin commented on content/en/resources/news.md:
```diff
@@ -0,0 +1,171 @@
+---
+title: News
+description: A listing of TUF news coverage.
```
```suggestion
description: TUF in the news
```

chalin commented on content/en/resources/_index.md:
```diff
@@ -1,7 +1,10 @@
 title: Resources
 menu: { main: { weight: 50 } }
-description: Learn more from our resources
+description:
```
```suggestion
```

chalin commented on content/en/resources/_index.md:
```diff
@@ -1,7 +1,10 @@
   type: docs
 ---
+
+Find curated selections of videos, press coverage, and publications designed to
```
You mention "press coverage" here.
Should there be a Press page under resources?

chalin commented on content/en/resources/news.md:
```diff
@@ -0,0 +1,171 @@
+description: A listing of TUF news coverage.
+---
+
+>
```
```suggestion
> The [press](/resources/press) page contains a full listing of news coverage.
```

chalin commented on README.md:
```diff
@@ -1,11 +1,11 @@
 [Docsy][] theme, hosted on [Netlify][].
 
-Preview for the development version: https://tuf-homepage.netlify.app/
+Preview for the development version: https://tufresources.netlify.app/
```
Could you set this to the main URL for this repo, rather than change it for each PR?

The PR's preview is available from the PR comments, see https://github.com/Dindihub/docsy-tuf.io/pull/12#issuecomment-2278346323, so it doesn't need to be added to the PR.

The same comment applies for line 3.

Dindihub replied:
Initially, I was having trouble setting it up for the main branch. When I did - https://tufio.netlify.app/ for some reason Netlify created one for the resources branch when I pushed. I'll take a look again. Thanks

chalin commented:
Hmm, just like https://github.com/cncf/techdocs/pull/272, this PR has 38 commits, which doesn't make sense. Again, I think that you need to rebase it, and then force push the changes.

Dindihub commented:
> Hmm, just like [cncf/techdocs#272](https://github.com/cncf/techdocs/pull/272), this PR has 38 commits, which doesn't make sense. Again, I think that you need to rebase it, and then force push the changes.

Noted. Thanks

chalin commented:
Thanks for creating a new PR. In the future, when closing and reopening a new PR, ensure that you address _all_ comments in the "old" PR, and mark comments with a thumbs up once you have addressed them. While you addressed the press-release page comment I made here in #13, my other comments were not addressed. For this time, I've repeated (and adjusted) them over in #13.

Dindihub commented:
> Thanks for creating a new PR. In the future, when closing and reopening a new PR, ensure that you address _all_ comments in the "old" PR, and mark comments with a thumbs up once you have addressed them. While you addressed the press-release page comment I made here in #13, my other comments were not addressed. For this time, I've repeated (and adjusted) them over in #13.



> Thanks for creating a new PR. In the future, when closing and reopening a new PR, ensure that you address _all_ comments in the "old" PR, and mark comments with a thumbs up once you have addressed them. While you addressed the press-release page comment I made here in #13, my other comments were not addressed. For this time, I've repeated (and adjusted) them over in #13.

@chalin Sorry, my mistake. Thanks for resubmitting.
